acceptance/bundle/integration_whl/base/script

```diff
@@ -0,0 +1,8 @@
+export SPARK_VERSION=$DEFAULT_SPARK_VERSION
+export PYTHON_WHEEL_WRAPPER=false
+envsubst < input.json.tmpl > input.json
+cat input.json
+$CLI bundle init . --config-file input.json
+trace $CLI bundle deploy
+trap "errcode trace '$CLI' bundle destroy --auto-approve" EXIT
```

Comment:
It's better to set the trap before the deploy, deploy might fail but will partially deploy resources and it would be best to clean them up in such cases as well

Reply:
Thanks, good point! Updated in https://github.com/databricks/cli/pull/2471/commits/b080fa8776c583108ca6d96ee47c0109b1847cda